lib/ferto/client.rb

```diff
@@ -56,12 +56,15 @@ def initialize(opts = {})
     #
     # @return [Ferto::Client::Response]
     def download(aggr_id:, aggr_limit: @aggr_limit, url:,
-                 callback_url:, mime_type: "", extra: {})
+                 callback_url: "", callback_dst: "",
+                 callback_type: Ferto::CALLBACK_HTTP, mime_type: "", extra: {})
```

Comment:
`Ferto::CALLBACK_HTTP` is not defined anywhere

Reply:
Good catch! While at it, do you think that we should have default values for `callback_{url,type,dst}`? I would say probably not.